code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -173,24 +171,18 @@
 		update_icon()
 		playsound(src, 'sound/machines/cryo_warning.ogg', volume) // Bug the doctors.
 		radio.talk_into(src, "Patient fully restored", radio_channel, get_spans(), get_default_language())
-		if(autoeject) // Eject if configured.
-			radio.talk_into(src, "Auto ejecting patient now", radio_channel, get_spans(), get_default_language())
-			open_machine()
 		return
 
 	var/datum/gas_mixture/air1 = AIR1
 
 	if(air1.gases.len)
-		if(mob_occupant.bodytemperature < T0C) // Sleepytime. Why? More cryo magic.
+		if(mob_occupant.bodytemperature < T0C)
 			mob_occupant.Sleeping((mob_occupant.bodytemperature / sleep_factor) * 2000)
 			mob_occupant.Unconscious((mob_occupant.bodytemperature / unconscious_factor) * 2000)
 		if(beaker)
-			if(reagent_transfer == 0) // Magically transfer reagents. Because cryo magic.
-				beaker.reagents.trans_to(occupant, 1, 10 * efficiency) // Transfer reagents, multiplied because cryo magic.
-				beaker.reagents.reaction(occupant, VAPOR)
-				air1.gases[/datum/gas/oxygen][MOLES] -= 2 / efficiency //Let's use gas for this
-			if(++reagent_transfer >= 10 * efficiency) // Throttle reagent transfer (higher efficiency will transfer the same amount but consume less from the beaker).
-				reagent_transfer = 0
+			beaker.reagents.trans_to(occupant, 1)

```

Comment:
this should probably still magically create reagents, just in much lower amounts.

Reply:
like, instead of 1:10 something like 2:5

Reply:
i'm iffy about that but eh, i guess we'll give it a try